tests/README.md

```diff
@@ -0,0 +1,41 @@
+# Testing
+
+## Requirement
+- nox (https://nox.thea.codes/en/stable/)
+- pyenv (https://github.com/pyenv/pyenv)
+- pytest (https://docs.pytest.org/en/stable/)
+
+## How this works: 
+1. Install `nox` and `pytest`
+```bash
+pip install -r requirements.txt
+```
+2. Run all Python versions configured in the `noxfile.py` file.
+
+```bash
+# Configure all enviroment with pyenv
+for version in 3.9 3.10 3.11 3.12 3.13; do pyenv install $version; done
```

Comment:
Maybe this whole block can be moved to a Makefile or something similar? And by running `make tests` it all happens automatically.